### PR TITLE
create user profile UI page

### PR DESCRIPTION
**Pull request by Balancedsan**

### CODE REVIEW

File: src/app/profile/user-profile/sidenav/sidenav.component.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      iconFamily:"fas",
+      icon:"wrench",
+      text:"Settings"
+    }
+  ]
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (XephyLon) [inline src/app/profile/user-profile/sidenav/sidenav.component.ts]: No point in keeping OnInit or the constructor

File: src/app/profile/user-profile/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                <h3>Dontkillmydog@gmail.com</h3>
+                <h3><a href="www.linkedin.com" target="_blank">Personal Site</a></h3>
+            </section>
+        </article>
+    </article>
+    <article class="showcase-items">
+        <section fxLayout="column" fxLayoutAlign="center center">
+            <h3>Showcase Items</h3>
+            <h1 class="showcase mt-8" fxLayout="row" fxLayoutAlign="center center">8</h1>
+        </section>
```
Comment (XephyLon) [inline src/app/profile/user-profile/header/header.component.html]: Maybe create a component out of that?

Comment (XephyLon) [reply src/app/profile/user-profile/header/header.component.html]: Unless it won't be used anywhere else or it won't be more than 3

Comment (Yejielw) [reply src/app/profile/user-profile/header/header.component.html]: to add to this, is a good practice to make a component for latter mantenability

**Pull request merged**